README.md

```diff
@@ -2,6 +2,22 @@
 
 This is a Promotheus Metrics exporter for Thoth.
 
+## Setting up Metrics Exporter locally.
+1. Create a `.env` file from the `.env.template`.
+2. Once you have populated all the values for `.env`, follow up with `pipenv install --dev`.
+3. oc login into the Openshift cluster. 
+5. Run the local version of thoth-storages after restoring the dump - [guide](https://github.com/thoth-station/storages#running-postgresql-locally) (or connect to the production db using replacing the env variables in `.env`).
+4. Run the metrics exporter using - `pipenv run python3 wsgi.py`
+5. You should see metrics in - [localhost:8080](http://localhost:8080)
+
+## Adding new metrics to be exported. 
+1. Add the metric you want to expose to [metrics.py](https://github.com/thoth-station/metrics-exporter/blob/master/thoth/metrics_exporter/metrics.py).
```

Comment:
Maybe mention that is based on Prometheus and therefore the metric type need to be from Prometheus available classes from Prometheus package installed (Gauge, Counter, Histogram) which has specific way to be set and specific value need to be provided.

Reply:
Thanks, I have added the link to Prometheus documentation :smile: